setup.py

```diff
@@ -1,6 +1,33 @@
+import subprocess
 import platform
 import os
 
+def write_to_file(file_name):
```

Comment:
seems like this function will run every time setup is ran. is it possible we can bring back the logic to do this conditionally? something like

```py
def alias_already_exists(path_to_config)
  with open(path_to_config, 'r') as f:
      for line in f.readlines():
        if 'export ESLINT_NO_DEV_ERRORS=true' in line:
          return True
  return False
```

Reply:
The last two commits should now resolve this.